### PR TITLE
Change Asset Class Creation Deposits

### PR DESCRIPTION
**Pull request by joepetrowski**
Change the asset class creation deposits to use standard storage deposit functionality.

Similar to https://github.com/polkadot-fellows/runtimes/pull/237

### CODE REVIEW

File: system-parachains/asset-hubs/asset-hub-polkadot/primitives/src/lib.rs [hunk 1]
```diff
@@ -52,7 +52,7 @@ frame_support::parameter_types! {
 	pub UncongestedMessage: Xcm<()> = build_congestion_message(false).into();
 
 	/// Should match the `AssetDeposit` of the `ForeignAssets` pallet on Asset Hub.
-	pub const CreateForeignAssetDeposit: u128 = 10 * UNITS;
+	pub const CreateForeignAssetDeposit: u128 = system_para_deposit(1, 190);
```
Comment (joepetrowski) [inline system-parachains/asset-hubs/asset-hub-polkadot/primitives/src/lib.rs:55]: cc @claravanstaden @acatangiu

Comment (shawntabrizi) [reply system-parachains/asset-hubs/asset-hub-polkadot/primitives/src/lib.rs:55]: 190 is based on the bytes? there should be some way to get that number without hardcoding no?

with `max_encoded_len`?

**Review by shawntabrizi — APPROVED** [review]

**Review by muharem — APPROVED** [review]

**Review by claravanstaden — APPROVED** [review]

File: system-parachains/asset-hubs/asset-hub-polkadot/primitives/src/lib.rs (hunk 1 above)
Comment (joepetrowski) [reply system-parachains/asset-hubs/asset-hub-polkadot/primitives/src/lib.rs:55]: It is the encoded length of [`AssetDetails`](https://github.com/paritytech/polkadot-sdk/blob/bc2e5e1/substrate/frame/assets/src/types.rs#L52) in this configuration.

I'm not sure if we can use `max_encoded_len` here? I calculated it by writing a test and asserting `encoded_len`.

Comment (bkchr) [reply system-parachains/asset-hubs/asset-hub-polkadot/primitives/src/lib.rs:55]: If you remove the const it should work.

**Review by bkchr — APPROVED** [review]

**Pull request merged**